genproto/pluginv2/backend.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.30.0
-// 	protoc        v3.21.12
+// 	protoc-gen-go v1.31.0
+// 	protoc        v4.24.3
```

Comment:
@marefr I reverted protoc, but I don't see any breaking changes here with protoc-gen-go:  https://github.com/protocolbuffers/protobuf-go/releases/tag/v1.31.0

Reply:
protoc-gen-go should be fine according to https://github.com/grafana/grafana-plugin-sdk-go/blob/main/contribute/developer-guide.md#generate-go-code-for-protobuf-definitions

Reply:
That documentation seems outdated and we'll try to fix that :) Still a bit confusing reading https://github.com/protocolbuffers/protobuf-go#compatibility, but sounds like there's a runtime version nowadays that should be compatible rather than the actual version of packages and if runtime version is not compatible it should create an error when using protoc-gen-go